### ISSUE TITLE
Improve compile-error feedback messages

### ISSUE THREAD

**Issue by monoti** [stryker-mutator/stryker4s#998]
### Summary

When I run `sbt stryker`, compilation fails due to it tries to mutate either `>=` or `<=` operation to `==`.
The correct syntax of Spark's condition for equal method is `===`.

I have prepared a repo to reproduce the issue: [stryker-spark-example](https://github.com/monoti/stryker-spark-example).

It will throw below error when you run `sbt stryker`.

```
[error] /Users/stryker-spark-example/target/stryker4s-572897741400037258/src/main/scala/SimpleDFHelper.scala:27:32: value or is not a member of Boolean
[error]         df.where($"col1" == 10 or col("col2") <= "B")
[error]                                ^
[error] /Users/stryker-spark-example/target/stryker4s-572897741400037258/src/main/scala/SimpleDFHelper.scala:35:47: type mismatch;
[error]  found   : Boolean
[error]  required: org.apache.spark.sql.Column
[error]         df.where($"col1" >= 10 or col("col2") == "B")
[error]                                               ^
[error] two errors found
[error] (Compile / compileIncremental) Compilation failed

```

### Stryker4s config

```hocon
stryker4s {
  mutate: [ "src/main/scala/*.scala"]
}
```

### Stryker4s environment

```
stryker4s 0.14.0
```

### Your Environment
| software             | version(s) |
| -------------------- | ---------- |
| Scala version        |    2.13.6        |
| Spark version        |    3.2.0        |
| Build tool & version |  1.3.13          | 
| Operating System     |    macOS Big Sur        |

Note that the issue occurs on Scala 2.12.x and Spark 2.4.x as well.

**Comment by monoti** [comment]
Hi @hugo-vrijswijk, 
Yes, it can continue. Thanks. Is there any plans to make Stryker mutation support Spark?

**Comment by hugo-vrijswijk** [comment]
@monoti I don't think that will happen anytime in the near future. Otherwise if we ever do type analysis for every mutant we'd have to check a possible mutation to both `==` and `===` which, in Scala, can come from a lot of different places. 

The mutant is marked as a compile error, but I think the sbt plugin should have better output so it doesn't show this as an error (it's not, it's just part of the process of removing non-compiling mutants)
